docs/Home.md

```diff
@@ -78,31 +90,87 @@ the file was created, you will upload it to Jenkins in a subsequent step:
     ```bash
     export CLUSTER=my-jenkins-cluster
     export ZONE=us-central1-c
-    gcloud container clusters create --issue-client-certificate --enable-legacy-authorizaton --zone $ZONE $CLUSTER
-    ```
-1. If using an existing cluster, enable legacy authorization on the cluster:
-    ```bash
-    gcloud container clusters update --enable-legacy-authorization --zone $ZONE $CLUSTER
+    gcloud container clusters create --zone $ZONE $CLUSTER
     ```
 1. Get credentials for the cluster:
     ```bash
     gcloud container clusters get-credentials --zone $ZONE $CLUSTER
     ```
 
-\***Note**: You can use an existing cluster but it must have been created with Client Certificates
-enabled. If creating your cluster through Cloud Console, click **Advanced Options** and then
-under **Security** make sure that "Issue a client certificate" is checked. This is a ***permanent***
-property of the cluster. The "Your first cluster" template, for example, does not have this checked
-by default so it won't be possible to use such a cluster. Also check "Enable legacy authorization",
-which is always disabled by default. From Cloud Console you can also select an existing cluster,
-click **EDIT** and change the **Legacy Authorization** dropdown to "Enabled".
-
 ### Configure Kubernetes Cluster Permissions
 
-1. Add the cluster-admin role to the service account group within your Kubernetes cluster:
+Your GCP service account will have limited IAM permissions. Use RBAC in kubernetes to configure permissions suited to your use case.
+
+#### Less Restrictive Permissions
+
+The following permissions will grant you full read and write permissions to your cluster.
+
+1. Add the cluster-admin role to the service account associated with your Kubernetes cluster:
     ```bash
-    kubectl create clusterrolebinding serviceaccounts-cluster-admin --group=system:serviceaccounts --clusterrole=cluster-admin
+	kubectl create clusterrolebinding cluster-admin-binding \
+    --clusterrole cluster-admin \
+    --user jenkins-gke@YOUR-PROJECT.iam.gserviceaccount.com
+    ```
+
+#### More Restrictive Permissions
+
+The following permissions will grant you enough permissions to deploy to your cluster.
+
+1. Grant your GCP login account cluster-admin permissions before creating the following roles/role bindings:
+	```bash
+	 kubectl create clusterrolebinding gcp-cluster-admin-binding --clusterrole=cluster-admin --user=my_gcp_login@google.com
+	```
+1. Create the ClusterRole yaml file:
+	```
+	apiVersion: rbac.authorization.k8s.io/v1beta1
+	kind: ClusterRole
+	metadata:
+     name: robot-deployer
+	rules:
```

Comment:
Indention needs to be properly formatted, this is especially important for YAML.

Reply:
Must've gotten butchered in the markdown. Should be fixed now that I just link to the files I used. Done.